pkg/manifest/storages/file_storage.go

```diff
@@ -0,0 +1,94 @@
+package storages
```

Comment:
`storages` is not a normal English usage. Maybe `storage`?

Reply:
Interesting, didn't think about, just used the terms used by the Amplitide package. Will change....